imap_processing/tests/idex/test_idex_l1b.py

```diff
@@ -0,0 +1,117 @@
+"""Tests the L1b processing for IDEX data"""
+
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.idex.idex_l1b import idex_l1b, unpack_instrument_settings
+
+
+@pytest.fixture()
+def l1b_dataset(decom_test_data: xr.Dataset) -> xr.Dataset:
+    """Return a ``xarray`` dataset containing test data.
+
+    Returns
+    -------
+    dataset : xr.Dataset
+        A ``xarray`` dataset containing the test data
+    """
+    dataset = idex_l1b(decom_test_data, data_version="001")
+    return dataset
+
+
+def test_l1b_cdf_filenames(l1b_dataset: xr.Dataset):
+    """Tests that the ``process_idex_l1b`` function generates datasets
```

Comment:
should this be this?
```suggestion
    """Tests that the ``idex_l1b`` function generates datasets
```